api/src/pcapi/routes/serialization/finance_serialize.py

```diff
@@ -53,3 +60,65 @@ def from_orm(cls, invoice: finance_models.Invoice) -> "InvoiceResponseModel":
 
 class InvoiceListResponseModel(BaseModel):
     __root__: list[InvoiceResponseModel]
+
+
+class LinkedVenues(BaseModel):
+    """A venue that is already linked to a bank account."""
+
+    id: int
+    common_name: str
+
+    class Config:
+        orm_mode = True
+
+
+class ManagedVenues(BaseModel):
+    id: int
+    common_name: str
+    siret: str
+    bankAccountId: int | None
+
+    class Config:
+        orm_mode = True
+
+    @classmethod
+    def from_orm(cls, venue: "offerers_models.Venue") -> "ManagedVenues":
+        """
+        For each managed venue, we only want to display the actual bank account
+        that is linked to it, if any.
+        """
+        if len(venue.bankAccountLinks) > 1:
+            raise AssertionError("There should be one or none bank account.")
+
+        if venue.bankAccountLinks:
+            venue.bankAccountId = venue.bankAccountLinks[0].bankAccountId
+        return super().from_orm(venue)
+
+
+class BankAccountResponseModel(BaseModel):
+    id: int
+    isActive: bool
+    label: str
+    obfuscated_iban: str
```

Comment:
Est-ce voulu de ne pas mettre en camel case comme les autres attributs ?
```suggestion
    obfuscatedIban: str
```